phase1/vsphere/README.md

```diff
@@ -48,16 +48,130 @@ cd kubernetes-anywhere
 ### Enter the `kubernetes-anywhere` deployment environment:
 
 ```shell
-make docker-dev
+make docker-dev # Building docker image for first time can take few minutes.
 ```
 
 ### Start the deployment wizard:
 
 ```shell
 make deploy
 ```
+
 and fill complete the config wizard to deploy a kubernetes-anywhere cluster.
 
+### Sample config
+
+* Select the number of nodes. Master + Number of nodes will be deployed.
+```
+number of nodes (phase1.num_nodes) [4] (NEW) 8
+```
+
+* Set the cluster name. A folder with the cluster name will be created to place all the VMs.
+```
+cluster name (phase1.cluster_name) [kubernetes] (NEW) k8s-test-cluster-1
+```
+
+* Select the provider, in this case it would be vsphere.
+```
+cloud provider: gce, azure or vsphere (phase1.cloud_provider) [gce] (NEW) vsphere
+```
+
+* Set the vCenter URL (Just the IP or domain name, without https://)
+```
+  vCenter URL Ex: 10.192.10.30 or myvcenter.io (phase1.vSphere.url) [] (NEW) 10.192.72.70
+```
+
+* Set the port for vCenter communication. Unless vCenter is setup with a different port select the default port.
+```
+  vCenter port (phase1.vSphere.port) [443] (NEW) 
+```
+
+* Enter the user name for vCenter. All vCenter operations will be performed using these user credentials
+```
+  vCenter username (phase1.vSphere.username) [] (NEW) administrator@myvcenter.io
+```
+
+* Enter the password for vCenter.
+```
+  vCenter password (phase1.vSphere.password) [] (NEW) MyPassword#3
+```
+
+* Set the type of certificate used by vCenter. Set to true for self signed certificates
+```
+  Does host use self-signed cert (phase1.vSphere.insecure) [true] (NEW) 
+```
+
+* Set the datacenter in vCenter to use. Specify the same datacenter to which the OVA was imported to.
+```
+  Datacenter (phase1.vSphere.datacenter) [datacenter] (NEW) PA-DC
+```
+
+* Set the datastore to be use. This will be used for placing the VMs and volumes created via storage classes/dynamic provisioning.
+```
+  Datastore (phase1.vSphere.datastore) [datastore] (NEW) vsanDatastore
+```
+
+* Specify a valid Cluster, Host or Resource Pool in which to deploy Kubernetes VMs. This should the same as the one selected when importing the template OVA. Example: Cluster: vsan-cluster, Host: 10.192.72.70 or Resource Pool: /vcqaDC/host/10.192.72.70/Resources or /vcqaDC/host/vsan-cluster/Resources
```

Comment:
Remove "in which". 
Resource Pool can be name of DRS RP right? without /vcqaDC/host/vsan-cluster